cartographer/mapping/internal/3d/local_trajectory_builder_3d.cc

```diff
@@ -54,6 +54,51 @@ LocalTrajectoryBuilder3D::LocalTrajectoryBuilder3D(
 
 LocalTrajectoryBuilder3D::~LocalTrajectoryBuilder3D() {}
 
+std::unique_ptr<transform::Rigid3d> LocalTrajectoryBuilder3D::ScanMatch(
+    const transform::Rigid3d& pose_prediction,
+    const sensor::PointCloud& low_resolution_point_cloud_in_tracking,
+    const sensor::PointCloud& high_resolution_point_cloud_in_tracking) {
+  if (low_resolution_point_cloud_in_tracking.empty()) {
```

Comment:
This is redundant. It is already verified before calling `ScanMatch()`. Remove similar to 2D?